shared/src/business/entities/User.ts

```diff
@@ -22,17 +37,7 @@ export class User extends JoiValidationEntity {
   public token?: string;
   public userId: string;
   public isUpdatingInformation?: boolean;
-  public contact?: {
-    address1: string;
-    address2?: string;
-    address3?: string;
-    city: string;
-    country: string;
-    countryType: string;
-    phone: string;
-    postalCode: string;
-    state: string;
-  };
+  public contact?: JudgeContact | NonJudgeContact;
```

Comment:
Instead of varying the contact based on user, I think it might be easier to create a new field called judgePhoneNumber. Then the schema for contact does not need to vary and it is clear in the future what fields should be ripped out of the User entity.